targets/bmv2/fuzzer.cpp

```diff
@@ -29,21 +29,19 @@ InitialP4RuntimeConfig Bmv2V1ModelFuzzer::produceInitialConfig() {
             continue;
         }
         auto table = tables.Get(tableId);
-        auto maxEntryGenCnt = table.size();
+        // TODO: remove this `min`. It is for ease of debugging now.
+        auto maxEntryGenCnt = std::min(table.size(), (int64_t)2);
```

Comment:
We should maybe have fuzzer options that are set by each target. They specify parameters like this. Separate from the command line options.

Reply:
Yeah agree. Added to TODO.